Makefile

```
@@ -331,9 +340,10 @@ go-version: ## Print the go version we use to compile our binaries and images
 ## --------------------------------------
 
 .PHONY: clean
-clean: ## Remove all temporary files and folders
+clean: ## Remove all temporary files,folders and tools
```

```suggestion
clean: ## Remove all temporary files, directories and tools
```
Missing space + folder is something on a desktop